test/10_test-build

```diff
@@ -15,10 +15,10 @@ cd "$(git rev-parse --show-toplevel)" || return
 
 # Test build:
 if command -v docker >/dev/null 2>&1; then
-  docker build -f "${1:-Dockerfile}" -t image .
+  docker build --no-cache -f "${1:-Dockerfile}" -t image .
 else
   echo " "
   echo "Did not find docker. Run this command:"
-  echo "docker build -f ${1:-Dockerfile} -t image ." image
+  echo "docker build -f ${1:-Dockerfile} --no-cache -t image ."
```

Comment:
Added `--no-cache` to `docker build`, so it'll pick up changes to dependencies like `requirements.txt`.